docker-compose.yml

```diff
@@ -1,36 +1,30 @@
+---
 # This Compose file is for development purposes.
 # For a production-ready version, see https://github.com/owdex/compose
-
-version: "3.8"
-
+version: "3"
 services:
-
   solr:
     container_name: solr
-    image: solr
-
+    image: docker.io/solr:9
     ports:
       - 8983:8983
     volumes:
       - ./data/solr/:/var/solr/data
       - ./misc/solr_configset:/owdex_configset:ro
       - ./misc/solr_entrypoint.sh:/entrypoint.sh
-
     entrypoint:
-    - /entrypoint.sh
-
+      - /entrypoint.sh
 
   mongo:
     container_name: mongo
-    image: mongo
-
+    image: docker.io/mongo
     volumes:
       - ./data/mongo:/data/db
 
   frontend:
     container_name: frontend
+    image: ghcr.io/owdex/owdex:develop
```

Comment:
This is confusing me -- we don't need to pull from an image here since this is being built locally for development.

Reply:
True, but is also used to label the images that is build localy.
so instead of many images like 
![image](https://user-images.githubusercontent.com/49623362/230893744-15f05481-f5d5-4a29-9a91-a3f2f88eaaea.png)

they now are called 
![image](https://user-images.githubusercontent.com/49623362/230893785-a1428296-f98d-4876-b7d2-f79ea44e7072.png)

Its more orderly, and easier to clean up that way

We should however agree on a tag that will be reserved for local dev use, that we never actually push to the registry.

Reply:
I'm pretty sure though that it's not possible for us to push to the registry ourselves, at least not without changing some things around to sign in to Docker on the CLI with our GitHub accounts somehow. I'm not sure how that would work, I only ever used the Actions build process.

Otherwise that makes sense. Thanks!